src/ensembl/production/core/clients/datachecks.py

```diff
@@ -39,6 +39,7 @@ def submit_job(self, server_url, dbname, species, division, db_type,
 
       email - optional address for an email on job completion
       tag - optional text for grouping datacheck submissions
+      target_url - determine which registry to use 
```

Comment:
This parameter isn't for determining the registry; it's used in addition to, and in preference to, the registry, for "ancillary" databases that are required for cross-checking when running datachecks on `server_url` (the set of dbs on which datachecks are run is derived from either `dbname`, or `db_type` + `species`/`division`.